packages/core/src/uploader/index.ts

```diff
@@ -7,29 +7,13 @@ export class Uploader {
     this.uploadEndpoint = uploadEndpoint
   }
 
-  public async publicUpload(
-    file: File | URL | string | undefined | null,
-  ): Promise<string | undefined | null> {
-    return this.upload(file, false)
-  }
-
-  public async privateUpload(
-    file: File | URL | string | undefined | null,
-  ): Promise<string | undefined | null> {
-    return this.upload(file, true)
-  }
-
-  private async upload(
-    file: File | URL | string | undefined | null,
-    isPrivate = false,
-  ) {
+  public async upload(file: File | URL | string | undefined | null) {
     if (file === null) return null
     if (!file) return
     if (file instanceof URL) return file.toString()
     if (typeof file === 'string') return file
     const formData = new FormData()
     formData.append('file', file)
-    formData.append('protected', isPrivate ? 'true' : 'false')
```

Comment:
@NicolasMahe could you confirm that removing this param will not have any unwanted side effects

Reply:
All good. This `protected` param is optional.